mupen64plus-core/src/pi/pi_controller.c

```diff
@@ -56,6 +58,11 @@ static void dma_pi_read(struct pi_controller *pi)
    uint32_t rom_address;
    const uint8_t* dram;
    uint8_t* rom;
+   
+   if( !AllowUnalignedDMA ) {
+	   pi->regs[PI_CART_ADDR_REG] &= 0xfffffffe;
```

Comment:
Actually writing to the register is a little ugly, but it works. (Just aligning it in a local variable here doesn't work. The register needs to actually be updated to be aligned)